crates/release_plz_core/src/git/github_graphql.rs

```diff
@@ -213,14 +208,6 @@ mod tests {
         // file removal
         fs_err::tokio::remove_file(&removed_path).await.unwrap();
 
-        // type change (replace symlink with a content it pointed to)
-        fs_err::tokio::remove_file(&type_changed_path)
-            .await
-            .unwrap();
-        fs_err::tokio::write(&type_changed_path, b"unchanged")
-            .await
-            .unwrap();
```

Comment:
why you changed the test? I expect the new implementation to be compatible with the previous one.